data-plane-framework/draft-ietf-detnet-data-plane-framework.xml

```diff
@@ -635,6 +657,22 @@ CE1----EN1--------R1-------R2-------R3--------EN2-----CE2
      prevent misordering of packets. In both cases the protection paths are
      established and maintained for the duration of the DetNet service.
      </t>
+   </section>
+     <section title="Path Differential Delay">
+             <t>
+     In the preceding example, reordering of packets is dependent on the number
+     of out-of-order packets that can be buffered and the delay difference of
+     arriving packets.  DetNet uses configuration of the number of maximum
+     number out-of-order packets and the maximum delay that out-of-order
+     packets can be held before being delivered. If the differential delay
+     between paths is large enough or the arrival rate of packets large enough
+     packets may be dropped instead of being reordered.  Likewise elimination
+     uses the sequence number to eliminate duplicate packets and large
+     differential delays combined with high numbers of packets may exceed the
+     ability of this elimination function to eliminate all duplicate packets.

```

Comment:
I think this is not valid. PEF works with a "seg.num window" and a "history". Packets with Seq.num out of the "window" are dropped. "Window" moves according to the highest received seq.num.  
Therefore PEF cannot result in scenarios where "... may exceed the ability of this elimination function to eliminate all duplicate packets". The worst case of PEF is to discard packets that should not be discarded.
Design of PEF parameters requires that "window" parameter is big enough to deal with the differential delay.

Reply:
[Don] A lot of this text comes from the TSN documents.  I checked what TSN says.   And in high speed scenarios a sequence number of 16 bits is not very long.  I think there are valid duplicate conditions.

[Balazs] This part of the TSN docu is not well written. The major problem of PEF in the here referred case is that slow path packets are dropped (they are out-of-window). Duplicate delivery may occur in very-very special cases. So proposed change:
"... may exceed the ability of this elimination function to work properly."